RecoEgamma/Configuration/python/RecoEgamma_cff.py

```diff
@@ -48,10 +48,19 @@
 from RecoHI.HiEgammaAlgos.photonIsolationHIProducer_cfi import photonIsolationHIProducerpp
 from RecoHI.HiEgammaAlgos.photonIsolationHIProducer_cfi import photonIsolationHIProducerppGED
 from RecoHI.HiEgammaAlgos.photonIsolationHIProducer_cfi import photonIsolationHIProducerppIsland
+from RecoHI.HiCentralityAlgos.HiCentrality_cfi import hiCentrality
+from RecoHI.HiCentralityAlgos.HiClusterCompatibility_cfi import hiClusterCompatibility
+pp_on_XeXe_2017.toModify(hiCentrality,
+                         producePixelTracks = False,
+                         srcTracks = cms.InputTag("generalTracks"),
+                         srcVertex = cms.InputTag("offlinePrimaryVertices")
+                         )
 
 _egammaHighLevelRecoPostPF_HI = egammaHighLevelRecoPostPF.copy()
 _egammaHighLevelRecoPostPF_HI += photonIsolationHIProducerpp
 _egammaHighLevelRecoPostPF_HI += photonIsolationHIProducerppGED
 _egammaHighLevelRecoPostPF_HI += photonIsolationHIProducerppIsland
+_egammaHighLevelRecoPostPF_HI += hiCentrality
```

Comment:
I wonder if egammaHighLevelRecoPostPF is the most appropriate place where to include hiCentrality and hiClusterCompatibility? Shouldn't they be more conveniently kept separated inside the highlevelreco sequence of /Configuration/StandardSequences/python/Reconstruction_cff.py

Reply:
Done in the new commit